concepts/loops/about.md

```diff
@@ -0,0 +1,59 @@
+# Loops
+
+Loops in Rust can be used to repeatedly execute a block of code until you tell it to stop or a certain condition is met.
+
+The most basic version of a loop uses the `loop` keyword:
+
+```rust
+fn main() {
+    loop {
+        println!("again!");
+    }
+}
+```
+
+The above code-snippet makes an infinite loop. To exit a loop use the break keyword:
+
+```rust
+fn main() {
+    let mut count = 0;
+    loop {
+        println!("again!");
+        count += 1;
+        if count > 10 { break ; }
+    }
+}
+```
+
+## Loops using `for` and `while`
+
+Apart from the `loop` keyword, you can also make loops with `for` and `while`.
+
+A `for` loop is used to iterate over the elements of a collection.
```

Comment:
```suggestion
A `for` loop is used to iterate over the elements of a collection or iterator.
```

Reply:
Might be worth adding a sentence explaining that ranges are iterators as well, so you can do things like `for i in 0..10 { ... }`.